test/Elastic.Apm.AspNetFullFramework.Tests/Elastic.Apm.AspNetFullFramework.Tests.csproj

```diff
@@ -9,19 +9,30 @@
     <!-- Always generate debug symbols this allows fluent symbols exception messages to include variable names -->
     <DebugSymbols>True</DebugSymbols>
   </PropertyGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(OS)' == 'WINDOWS_NT'">
+    <PackageReference Include="Microsoft.AspNetCore.App" />
```

Comment:
Could you please clarify why do we need `Microsoft.AspNetCore.App` dependency?

Reply:
https://github.com/dotnet/sdk/issues/2253 

This is too squash compiler warnings since this test project does not end up pulling these in.